test/unit-tests/expression/parse.test.js

```diff
@@ -1336,11 +1336,16 @@ describe('parse', function () {
 
     it('should follow precedence rules for implicit multiplication and division', function () {
       assert.strictEqual(parseAndStringifyWithParens('2 / 3 x'), '(2 / 3) x')
+      assert.strictEqual(parseAndStringifyWithParens('-2/3x'), '((-2) / 3) x')
```

Comment:
@gwhitney are there also tests in place to check how non-constant nodes in numerator/denominator are handled now? Like:  `-(2+3)/3x`, `-2/(3+4)x`, etc. I.e. the cases you listed in https://github.com/josdejong/mathjs/issues/2370#issuecomment-1031077094

Reply:
OK, have added any more test cases in both parse.test.js and OperatorNode.test.js.